vars/neoDeploy.groovy

```diff
@@ -216,7 +231,73 @@ void call(parameters = [:]) {
                 lock("$STEP_NAME:${neoCommandHelper.resourceLock()}") {
                     deploy(script, configuration, neoCommandHelper, configuration.dockerImage, deployMode)
                 }
+                inValidateCache(configuration)
+            }
+        }
+    }
+}
+
+private inValidateCache(configuration){
+    if(configuration.neo.invalidateCache == true) {
```

Comment:
What about having this check outside the method? Strange to have a method `invalidateCache` which does not invalidated based on some config settings.

Reply:
Good point 👍